language/move-prover/bytecode/src/stackless_bytecode.rs

```diff
@@ -173,6 +224,7 @@ impl BorrowNode {
 pub enum PropKind {
     Assert,
     Assume,
+    Modifies,
```

Comment:
Is this the statement for "havoc"?  If yes, what is the argument to this statement?

Reply:
Currently I'm passing in the same expression as we have on spec lang level (global<T>(a))